jrcc-access-spring-boot-autoconfigure/src/test/java/ca/bc/gov/open/jrccaccess/autoconfigure/plugins/http/DocumentControllerTests.java

```diff
@@ -92,13 +108,10 @@ public void post_with_sevice_unavailable_input_should_return_503_response() {
 	@Test
 	public void post_with_io_exception_input_should_return_500_response() {
 		@SuppressWarnings("rawtypes")
-		ResponseEntity response = sut.postDocument(VALID, null, null, null, null, null, this.multipartFileWithException);
+		ResponseEntity response = sut.postDocument(null, null, null, null, null, this.multipartFileWithException);
 
 		assertEquals(HttpStatus.INTERNAL_SERVER_ERROR, response.getStatusCode());
 		assertEquals(HttpStatus.INTERNAL_SERVER_ERROR.getReasonPhrase(), ((ca.bc.gov.open.api.model.Error)response.getBody()).getMessage());
 		assertEquals(Integer.toString(HttpStatus.INTERNAL_SERVER_ERROR.value()), ((ca.bc.gov.open.api.model.Error)response.getBody()).getCode());
 	}
-
-	//@Test
```

Comment:
this seemed like code that was not meant to be around so I removed it, please lmk if you would like it to be included as is 😄

Reply:
Yup, good clean up, rule of thumb, if code commented: delete